pkg/controllers/testutil/zcmd/zpool/add.go

```diff
@@ -0,0 +1,89 @@
+/*
+Copyright 2020 The OpenEBS Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package zpool
+
+import (
+	"fmt"
+	"math/rand"
+	"strings"
+	"time"
+
+	cstor "github.com/openebs/api/pkg/apis/cstor/v1"
+	"github.com/pkg/errors"
+)
+
+// Add mocks the zpool add command and returns error based on the test configuration
+func (mPoolInfo *MockPoolInfo) Add(cmd string) ([]byte, error) {
+	if mPoolInfo.PoolName == "" {
+		return []byte("cannot open 'pool': no such pool"), errors.Errorf("exist status 1")
+	}
+	// If configuration expects error then return error
+	if mPoolInfo.TestConfig.ZpoolCommand.ZpoolAddError {
+		return addError(cmd)
+	}
+	mPoolInfo.addVdev(cmd)
+	return []byte{}, nil
+}
+
```

Comment:
n_p: typo `exist` --->`exit`

Reply:
Also in practice does zpool command returns the string `exit status 1` as string ?